lib/purge.js

```diff
@@ -0,0 +1,71 @@
+const axios = require('axios')
+const creds = require('./config')
+const fs = require('fs')
+const urljoin = require('url-join')
+
+module.exports.purge = async() => {
+  const config = creds.readConfig()
+  if (!checkDpIsThere()) {
+    console.error('Aborting: datapckage.json not found!')
+    process.exit(1)
+  }
+  const dpjson = JSON.parse(fs.readFileSync('datapackage.json').toString())
+  const token = await getToken(config)
+  const resp = await requestPurge(config, dpjson.name, token)
+  if (resp.status === 'OK') {
+    console.log(`Successfully deleted ${dpjson.name}`)
+  } else {
+    console.log(`Failed to deleted ${dpjson.name}`)
+  }
+}
+
+const getToken = async(config) => {
```

Comment:
It seems like we have these functions in our branch for `push` command, so in the future, we need to have them somewhere in the `utils` and import them as needed to be as DRY as possible.

Probably, we could start doing it from here?